buildimage.go

```diff
@@ -34,23 +36,31 @@ func init() {
 func extraBinMust(n string) string {
 	p, err := exec.LookPath(n)
 	if err != nil {
-		log.Fatalf("extraMustBin(%q): %v; you may need to run sudo apt install wireless-tools wpasupplicant", n, err)
+		log.Fatalf("extraMustBin(%q): %v", n, err)
 	}
 	return p
 }
 func main() {
+	var args = []string{
+		"go", "run", "github.com/u-root/u-root/.",
+		"-files", extraBinMust("iwconfig"),
+		"-files", extraBinMust("iwlist"),
+		"-files", extraBinMust("wpa_supplicant"),
+		"-files", extraBinMust("wpa_action"),
+		"-files", extraBinMust("wpa_cli"),
+		"-files", extraBinMust("wpa_passphrase"),
+	}
+	if *bzImage != "" {
+		args = append(args, "-files", *bzImage+":bzImage")
```

Comment:
why path bzImage here, add`file := filepath.Join(user.home, "linux/arch/x86/boot/bzImage:bzImage)` or something that accomplishes that, then on line 44, make it this `"go", "run", "github.com/u-root/u-root/.", "file"`

Reply:
no, look at the args []string above. If you look at where it use to be you should be able to see I moved it, so I can add to it based on the bzImage and iso flags.

The args are used in the commands [][]string below. You can see how it used to be done. the commands array is what is used in the range commands for loop below. The args have to be the last thing run, after the go get commands.

Plus with what you are suggesting, we would not get all the other -files commands we want. 

Further, you only want to run u-root once

 ... and with your file := suggestion you would be forcing people to only be able to ever use $HOME/linux/arch/x86/boot/bzImage as their kernel, which is very inconvenient.

You should fetch this and try it out and you'll see how it works.